### ISSUE TITLE
Review use of website backlog page

### ISSUE THREAD

**Issue by kellylee-gds** [alphagov/govuk-design-system#1876]
## What
Understand how the [backlog page](https://design-system.service.gov.uk/community/backlog/) on the Design System website is being used, with a view to retire if not visited.

## Why
The backlog page on the website is out of date with the GitHub community backlog.

## Who needs to know about this
Charlotte, Kelly, Senior Performance Analyst 
If we choose to deprecate - Developer, Content Designer

## Done when
- [ ] Use GA to understand traffic to the backlog page
- [ ] Agree whether to retire or remain

## Next steps
[outcome of decision]

**Comment by kellylee-gds** [comment]
@christopherthomasdesign @CharlotteDowns @trang-erskine I had a look at historical server logs data and average fortnightly page views of the website backlog page were around 678

**Comment by SaraC19** [comment]
Just adding here what I sent to Kelly in Slack:

So a few things (data from 17th Aug - 20th Sept 2021):
(Note that I think some of this data is a little skewed from the community events we've had this month!)

- It doesn't get a super high volume of traffic but it is a consistent amount (report attached)
- Most of the traffic comes from /community (which is in the top navigation bar)
- It's the most clicked on page for users that visit /community
- Once a user is there, there are a lot of clicks off into GitHub (report attached)

I don't think the page is perfect, but it does suggest people are engaging with it and interested

[GitHub clicks from the backlog](https://github.com/alphagov/govuk-design-system/files/7211946/Analytics.DS_.All.Web.Site.Data.Top.Events.20210817-20210920.1.pdf)

[Traffic summary](https://github.com/alphagov/govuk-design-system/files/7211943/Analytics.DS_.All.Web.Site.Data.Pages.20210817-20210920.2.pdf)

**Comment by christopherthomasdesign** [comment]
I'd be tempted to keep the page, but remove the long list of backlog items there. Seeing as they duplicate what's on GitHub and need to be manually kept up-to-date with GitHub. And probably make the link to the GitHub slightly more prominent. 

Depending on how we re-structure the backlog, we might want to revisit the content of the page more generally.

**Comment by trang-erskine** [comment]
Thanks for sharing the data Sara. So interesting that access denied pattern is the most clicked on backlog item https://github.com/alphagov/govuk-design-system-backlog/issues/158. The next 2, filter a list and pagination is expected. 

I'm also in favour of Chris's suggestion to remove the list.

**Comment by christopherthomasdesign** [comment]
My guess is 'access denied' is the most clicked on because it's top of the list 😆

**Comment by SaraC19** [comment]
Yeah Chris is right - you do generally see that the first thing in a list is always clicked on the most (on GOV.UK especially). Removing the list sounds like a good idea!
